profiles/common/modules/features/nexteuropa_trackedchanges/src/NETCInfoController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @file
+ * Contains Drupal\nexteuropa_trackedchanges\NETCInfoController.
+ */
+
+namespace Drupal\nexteuropa_trackedchanges;
+
+/**
+ * Class NETCInfoController.
+ */
+class NETCInfoController extends \EntityAPIController {
+
+  /**
+   * Override the save method.
+   *
+   * It sets the Scan date and the label (if not set) before saving.
+   */
+  public function save($entity, DatabaseTransaction $transaction = NULL) {
+    if (empty($entity->rel_entity_type)) {
+      throw new Exception('NextEuropaTrackedChangesInfo entity_type missing');
+    }
+
+    if (empty($entity->rel_entity_id)) {
+      throw new EntityMalformedException('NextEuropaTrackedChangesInfo entity_id missing');
+    }
+
+    // Forced data setting.
+    $entity->scanned = REQUEST_TIME;
```

Comment:
Same as before: I followed same logic as in other contrib modules. 

Furthermore, if I do that, I must do it for other attributes and I get camel lower case error messages on phpcs